### PR TITLE
cleanup unused vars

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- just want to cleanup unused vars from linter

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by yangwao** [comment]
hey @preschian feel free put you address up there for payout :)

**Review by roiLeo — APPROVED** [review]
looks good, 31 fixed issues

File: utils/proxy.ts [hunk 1]
```diff
@@ -2,31 +2,14 @@ import Axios from 'axios'
...
-type CdnUploadResponse = {
-  result: {
-    id: string
-    filename: string
-    uploaded: string
-    requireSignedURLs: boolean
-    variants: string[]
-  }
-  success: boolean
-}
```
Comment (vikiival) [inline utils/proxy.ts:21]: This is my only concern 👁️

File: components/unique/Create/CreateToken.vue [hunk 2]
```diff
@@ -147,7 +147,7 @@ export default class CreateToken extends mixins(
       data: { collectionEntities },
     } = collections
 
-    this.collections = collectionEntities.nodes?.map((ce: any) => ({
+    this.collections = collectionEntities.nodes?.map((ce) => ({
```
Comment (vikiival) [inline components/unique/Create/CreateToken.vue:150]: Can we have a type?

File: utils/proxy.ts (hunk 1 above)
Comment (preschian) [reply utils/proxy.ts:21]: can you elaborate more about this line? it seems like no file used that type. I found similar type in here https://github.com/kodadot/nft-gallery/blob/main/utils/directUpload.ts#L20-L29

File: components/unique/Create/CreateToken.vue (hunk 2 above)
Comment (preschian) [reply components/unique/Create/CreateToken.vue:150]: I think I will revert that `any` stuff for now, need more observation each files on `any` types

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 154.93 USD/KSM ~ 0.645 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0xd7b8469799d25852c9932f20e6b9dc1faead25ae9ae081c03bf83c2161758e63](https://kusama.subscan.io/extrinsic/0xd7b8469799d25852c9932f20e6b9dc1faead25ae9ae081c03bf83c2161758e63)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by preschian** [comment]
> 😍 Perfect, I’ve sent the payout 💵 $100 @ 154.93 USD/KSM ~ 0.645 $KSM 🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM) 🔗 [0xd7b8469799d25852c9932f20e6b9dc1faead25ae9ae081c03bf83c2161758e63](https://kusama.subscan.io/extrinsic/0xd7b8469799d25852c9932f20e6b9dc1faead25ae9ae081c03bf83c2161758e63)
> 
> 🪅 Let’s grab another issue and get rewarded! 🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

@yangwao seems like fail? cmiiw
this one also https://github.com/kodadot/nft-gallery/pull/2611

**Pull request merged**